supertokens_python/recipe/session/cookie_and_header.py

```diff
@@ -316,22 +327,24 @@ def access_token_mutator(
     def mutator(
         response: BaseResponse,
     ):
-        _set_access_token_in_response(
-            response, access_token, front_token, config, transfer_method
+        _ = _set_access_token_in_response(
+            request, response, access_token, front_token, config, transfer_method
         )
 
     return mutator
 
 
-def _set_access_token_in_response(
+async def _set_access_token_in_response(
+    req: BaseRequest,
     res: BaseResponse,
     access_token: str,
     front_token: str,
     config: SessionConfig,
     transfer_method: TokenTransferMethod,
 ):
     _set_front_token_in_headers(res, front_token)
-    _set_token(
+    await _set_token(
+        req,
         res,
         config,
         "access",
```

Comment:
I kinda don't like the fact that some private (`_`) functions in this file are now async while others are still sync. It's very easy to do  mistakes by missing await. Can we do something about this? 

Maybe add `async` after the function name?